.github/workflows/releaseConfigurator.yaml

```diff
@@ -0,0 +1,64 @@
+name: Configurator Release
+
+# This workflow manually triggers the automated process of release based on the changelog.
+# Is recommended to run `make release-changelog` before to see an example of what is going to be generated.
+
+# IMPORTANT No matter the branch selected when triggered, the workflow will always execute on 'main' branch.
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    name: Create release
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.version.outputs.next-version }}
+      image-tag: ${{ steps.image-tag.outputs.tag }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: main
+      - uses: newrelic/release-toolkit/validate-markdown@v1
+      - uses: newrelic/release-toolkit/generate-yaml@v1
+        with:
+          excluded-dirs: charts,.github
+      - uses: newrelic/release-toolkit/is-held@v1
```

Comment:
Right now the is-held action, until I create the fix later today, is not having the possibility to pass a flag the command has called fail. When fail is set to true, the command will exit with a 1.
So, without this flag, the is-held can only be treated by checking the output of the action and acting whether is true or false.
I think failing with an exit 1 is the way to go.

Reply:
i will remove this, since it looks like more for an automated flow than this manual triggered release

Reply:
Yes, I agree.